docs/treants.rst

```diff
@@ -58,32 +59,8 @@ will be reflected in the Treant in the other session ::
     <Tags(['elm'])>
 
 This is because both objects pull their identifying information from the same
-file on disk; they store almost nothing in memory.
-
-.. note:: The ``uuid`` of the Treant in this example will certainly differ from
-          any Treants you generate. This is used to differentiate Treants
-          from each other. Unexpected and broken behavior will result from
-          changing the names of state files!
-
-What goes into a state file?
-============================
-The state file of a Treant contains the core pieces of information that define
-it. A few of these things are defined in the filesystem itself, including ::
-
-    /home/bob/research/arborea/sprout/Treant.2b4b5800-48a7-4814-ba6d-1e631a09a199.json
-    |_________________________|______|______|____________________________________|
-              location          name     ^                 uuid 
-    |________________________________|   |
-                abspath               treanttype
-    |_________________________________________________________________________________|
-                                      filepath
-
-This means that changing the location or name of a Treant can be done at the
-filesystem level. Although this means that one can change the treanttype and
-uuid as well, this is generally not recommended.
+place on disk; they store almost nothing in memory.
```

Comment:
Please also point out that we only keep the filehandle alive as shortly as possible. It's good on rare occasions to have this documented.

Reply:
Good call. Added.